src/applications/simple-forms/21-0845/components/GroupCheckboxWidget.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react';
```

Comment:
this is the same as the one from 21-10210 right? what do you think of moving this to our simple-forms/shared/components and using in both places?

Reply:
Moved.  I'll open a follow-up PR for 21-10210's move soon.